app/Main.hs

```diff
@@ -518,6 +518,15 @@ addFromBower name = do
     stripBowerNamePrefix s = fromMaybe s $ T.stripPrefix "purescript-" s
     mkPackageName' = Bifunctor.first show . mkPackageName . stripBowerNamePrefix
 
+formatPackageFile :: IO ()
+formatPackageFile = do
+    pkg <- readPackageFile
```

Comment:
Totally up to you, but I don't think `pkg` and `db` are great names here, how about doing it pointfree?

`readPackageFile >>= readPackageSet >>= writePackageSet`

Reply:
Sure, just mirrors some other parts but that's fine for me